boa_engine/fuzz/fuzz_targets/common.rs

```diff
@@ -0,0 +1,71 @@
+use boa_engine::context::Context;
+use boa_engine::syntax::ast::visitor::{VisitWith, VisitorMut};
+use boa_engine::syntax::ast::{Expression, StatementList};
+use boa_interner::Sym;
+use libfuzzer_sys::arbitrary;
+use libfuzzer_sys::arbitrary::{Arbitrary, Unstructured};
+use std::fmt::{Debug, Formatter};
+use std::ops::ControlFlow;
+
+pub struct FuzzData {
+    pub context: Context,
+    pub ast: StatementList,
+}
```

Comment:
Even if the usage of the structure could be considered straightforward, adding some documentation could be useful.